config/ignition_citadel_gazebo11_ubuntu_focal.yaml

```diff
@@ -0,0 +1,6 @@
-tools4) | Package (% libignition-fuel-tools4-dev) | Package (% libignition-gazebo3) | Package (% libignition-gazebo3-dev) | Package (% libignition-gazebo3-plugins) | Package (% libignition-gui3) | Package (% libignition-gui3-dev) | Package (% libignition-launch2) | Package (% libignition-launch2-dev) | Package (% libignition-math6) | Package (% libignition-math6-dbg) | Package (% libignition-math6-dev) | Package (% libignition-math6-eigen3-dev) | Package (% libignition-msgs5) | Package (% libignition-msgs5-dbg) | Package (% libignition-msgs5-dev) | Package (% libignition-physics2) | Package (% libignition-physics2-core-dev) | Package (% libignition-physics2-dartsim) | Package (% libignition-physics2-dartsim-dev) | Package (% libignition-physics2-dev) | Package (% libignition-physics2-mesh-dev) | Package (% libignition-physics2-sdf-dev) | Package (% libignition-plugin) | Package (% libignition-plugin-dev) | Package (% libignition-rendering3) | Package (% libignition-rendering3-core-dev) | Package (% libignition-rendering3-dev) | Package (% libignition-rendering3-ogre1) | Package (% libignition-rendering3-ogre1-dev) | Package (% libignition-rendering3-ogre2) | Package (% libignition-rendering3-ogre2-dev) | Package (% libignition-sensors3) | Package (% libignition-sensors3-air-pressure) | Package (% libignition-sensors3-air-pressure-dev) | Package (% libignition-sensors3-altimeter) | Package (% libignition-sensors3-altimeter-dev) | Package (% libignition-sensors3-camera) | Package (% libignition-sensors3-camera-dev) | Package (% libignition-sensors3-core-dev) | Package (% libignition-sensors3-depth-camera) | Package (% libignition-sensors3-depth-camera-dev) | Package (% libignition-sensors3-dev) | Package (% libignition-sensors3-gpu-lidar) | Package (% libignition-sensors3-gpu-lidar-dev) | Package (% libignition-sensors3-imu) | Package (% libignition-sensors3-imu-dev) | Package (% libignition-sensors3-lidar) | Package (% libignition-sensors3-lidar-dev) | Package (% libignition-sensors3-logical-camera) | Package (% libignition-sensors3-logical-camera-dev) | Package (% libignition-sensors3-magnetometer) | Package (% libignition-sensors3-magnetometer-dev) | Package (% libignition-sensors3-rendering) | Package (% libignition-sensors3-rendering-dev) | Package (% libignition-sensors3-rgbd-camera) | Package (% libignition-sensors3-rgbd-camera-dev) | Package (% libignition-sensors3-thermal-camera) | Package (% libignition-sensors3-thermal-camera-dev) | Package (% libignition-tools-dev) | Package (% libignition-transport8) | Package (% libignition-transport8-core-dev) | Package (% libignition-transport8-dbg) | Package (% libignition-transport8-dev) | Package (% libignition-transport8-log) | Package (% libignition-transport8-log-dev) | Package (% libogre-2.1) | Package (% libogre-2.1-dev) | Package (% libsdformat9) | Package (% libsdformat9-dbg) | Package (% libsdformat9-dev) | Package (% sdformat9-dbg) | Package (% sdformat9-doc) | Package (% sdformat9-sdf)
```

Comment:
cmake2 is available from Ubuntu repositories. We don't need to import it.

Reply:
Because it's not present in the upstream (packages.osrfoundation.org) repository and because the filter won't fail if one of the items is not found, I'm super torn between keeping the filter_formula the same between the Debian and Ubuntu to make upkeep of this list easier and not having a package which you rightfully identify as spurious in the list here.

Reply:
Removed in 68bfa05